.github/CODEOWNERS

```diff
@@ -16,13 +26,32 @@
 /scripts/ @apb7
 
 
+# Exploration project.
+/core/controllers/editor*.py @DubeySandeep
```

Comment:
@seanlip, I have taken the part which I can review confidently and added aks681 for other so that we shouldn't have lots of reviewers of a single PR (Also, Akshay is also owning the new struct project, so maybe this will help him to be overwhelmed by PR reviews.) Let me know if you want me to add bot of us for each part?

Reply:
I think one is better actually. So, what you have sounds good to me.